project/infra/code-index/src/api.py

```
@@ -0,0 +1,212 @@
+    if not _db_client:
+        raise HTTPException(status_code=503, detail="Service not initialized")
+
+    indexer = CodeIndexer(arcadedb_client=_db_client, embedder=_embedder)
+    return indexer.index(request)
+
+
+if __name__ == "__main__":
+    import uvicorn
+
+    port = int(os.getenv("CODE_INDEX_PORT", "8080"))
+    host = os.getenv("CODE_INDEX_HOST", "0.0.0.0")
```

Ruff (0.15.2)

[error] Possible binding to all interfaces

(S104)